yjit/src/options.rs

```diff
@@ -30,6 +30,9 @@ pub struct Options {
     /// Dump compiled and executed instructions for debugging
     pub dump_insns: bool,
 
+    /// Print when specific ISEQ items are compiled or invalidated
+    pub dump_iseqs: Option<&'static str>,
```

Comment:
Maybe we also want to rename this option to `--dump-iseq-disasm` instead?

Reply:
get_option!().clone() still falls afoul of no copying:

~~~
warning: unused macro definition: `c_callable`
   --> src/utils.rs:183:14
    |
183 | macro_rules! c_callable {
    |              ^^^^^^^^^^
    |
    = note: `#[warn(unused_macros)]` on by default

error[E0204]: the trait `Copy` may not be implemented for this type
  --> src/options.rs:4:10
   |
4  | #[derive(Copy, Clone, PartialEq, Eq, Debug)]
   |          ^^^^
...
34 |     pub dump_iseq_disasm: Option<String>,
   |     ------------------------------------ this field does not implement `Copy`
   |
   = note: this error originates in the derive macro `Copy` (in Nightly builds, run with -Z macro-backtrace for more info)
~~~

I tried making a variant on get_option that cloned, but it didn't immediately work. There may be a way to make it work, though. I'll have another go.

Reply:
Okay, yeah, get_option!().clone() still has the same problem.

I'll see if there's some way to make a variant of get_option() that clones successfully. It seems like it *should* be possible...

~~~
error[E0507]: cannot move out of `OPTIONS.dump_iseq_disasm` as `OPTIONS` is a static item
    --> src/options.rs:66:18
     |
66   |         unsafe { OPTIONS.$option_name }
     |                  ^^^^^^^^^^^^^^^^^^^^ move occurs because `OPTIONS.dump_iseq_disasm` has type `Option<String>`, which does not implement the `Copy` trait
     |
    ::: src/core.rs:1381:31
     |
1381 |         if let Some(substr) = get_option!(dump_iseq_disasm).clone() {
     |                               ----------------------------- in this macro invocation
     |
     = note: this error originates in the macro `get_option` (in Nightly builds, run with -Z macro-backtrace for more info)
~~~

Reply:
Okay, I think I figured it out. You can take a reference, then call Option::as_ref(), then unwrap it. That's working for me with a second macro, which I may be able to do without.